### ISSUE TITLE
Experiments in model (webapp)

### ISSUE THREAD

**Issue by David-Taub** [allegroai/clearml#810]
## Proposal Summary
When looking at a model in the webapp, I'd like to be able to see the experiments that used this model as an input. Preferably, this should look just like the "creating experiment" row in General tab, but with multiple links to the experiments that used this model as an input.

## Motivation
When a model is retrained or evaluated, it is hard to find exactly in what experiment it happened.

**Comment by ainoam** [comment]
Thanks for suggesting this @David-Taub. Makes a lot of sense.
Model pages should be getting some added content in the next few months. We'll add this to the list :)

**Comment by ainoam** [comment]
@David-Taub What additional experiment information apart from its name would you find helpful here (e.g. tags)?

**Comment by David-Taub** [comment]
name (+link to experiment), id, type, tags, date, iterations, status
bonus: filterable & sortable list

**Comment by ainoam** [comment]
Thanks for the input @David-Taub.
Name and tags make perfect sense. For the rest, they seem to be a little superfluous for viewing in this specific case: a list of experiments which make use of this model.
Can you share some context on the use-cases in which you'd find the other fields you mention useful?

**Comment by David-Taub** [comment]
**TL;DR:**
the rest would make it easy to browse through a very big list of experiments.
**Use-case:**
You have a trained model. You've ran 15 different benchmarks for this model with different configs, half of the benchmarks failed and half worked as expected. After the benchmarking, you've decided that the model is good and you finetuned the model to 12 specific tasks, again some of the finetuning failed and some didn't, some had better results than other, some were done a long time ago and some are more relevant.

Without further information, you only have a list of 27 experiments names to search the one experiment you need. The user needs to understand which experiemnts are more relevant, which failed, which converged to useful new model, etc. Without giving further info about the experiments, the user needs to either:

1. click on all experiemnts and understand which one is needed
2. change the experiments names retroactively with all the relevant info (e.g. did the experiment failed or not)
3. add multiple tags that replicate the relevant info and use the tags to find the needed experiment

**Comment by ainoam** [comment]
Thanks for the detailed description @David-Taub - Picture is much clearer.

Sounds like this use case would be better served by using an input model as a filter on the experiment table, than by pushing the whole table into the model view. WDYT?

**Comment by David-Taub** [comment]
Sounds good. 
This is a pain fo us, hope this gets prioritised

**Comment by ainoam** [comment]
@David-Taub So the next ClearML server version would add a list of "used by" experiments to models, as in the described when the issue was opened, while providing models as filters for the experiment table will be added to the workplan for later down the road.

**Comment by pollfly** [comment]
Hey @David-Taub! [clearml-server v1.10](https://github.com/allegroai/clearml-server/releases/tag/v1.10) is now out with a "Lineage" tab in the UI model page, which displays the experiment that created the mode and lists experiments that use the model

**Comment by jkhenning** [comment]
@David-Taub closing this, please reopen if required.
